packages/api/src/utils/billing.js

```diff
@@ -85,10 +100,16 @@ export function createMockUserCustomerService () {
   const upsertUserCustomer = async (userId, customerId) => {
     userIdToCustomerId.set(userId, customerId)
   }
+
+  const createUserTosAgreement = async (userId, agreement) => {
+    // noop
```

Comment:
for my own sanity, are these needed?

Reply:
It's possible I'm bloating the CustomerService type.  Maybe in general Terms of Service should be handled in its own type, since it doesn't really have anything to do with the stripe customer, and it's really not even strictly a billing thing in general (although we're applying them to only billing at the moment).  I'll consider a quick refactor

Reply:
refactor done